cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -14,25 +14,38 @@
 # limitations under the License.
 
 
-# Configures etcd related flags of kube-apiserver.
+# Configures etcd related parameters of kube-apiserver.
 function configure-etcd-params {
   local -n params_ref=$1
 
+  local host_ip="127.0.0.1"
+  # If etcd is configured to listen on host IP,
+  # host_ip is set to the primary internal IP of host VM.
+  if [[ ${ETCD_LISTEN_ON_HOST_IP:-} == "true" ]] ; then
+      if [[ -n ${HOST_PRIMARY_IP:-} ]]; then
+          host_ip=$HOST_PRIMARY_IP
+      else
+          host_ip=$(hostname -i)
+      fi
```

Comment:
why not just:

host_ip="${HOST_PARIMATY_IP:-$(hostname -i)}

?

Reply:
Good suggestion. Updated.